server/application/application.proto

```diff
@@ -50,6 +50,19 @@ message ApplicationDeleteRequest {
 	optional bool cascade = 2;
 }
 
+// ApplicationManifestDiffRequest is a request to compare two manifests
+message ApplicationManifestDiffRequest {
+	required string appName = 1;
+	optional string resourceName = 2;
+	optional string apiVersion = 3;
+	optional string kind = 4;
+}
+
+// ApplicationManifestDiffResponse is the response to the comparison of two manifests
+message ApplicationManifestDiffResponse {
+	repeated string diffs = 1;
```

Comment:
@alexmt do we need more structure around ApplicationManifestDiffResponse? Is a list of arbitrary ordered strings okay?